src/client/components/ProfileFormComponent/ProfileFormComponent.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;600&family=Roboto:wght@300&display=swap');
+
+.avatar-Image img {
+  padding: 10px 40px;
+  display: flex;
+  margin: 40px auto;
+  justify-content: center;
+  width: 280px;
+  height: 150px;
+  background-color: #f5f5f5;
+  border-radius: 10px;
+}
+
+figcaption {
+  text-align: center;
+  font-size: 20px;
+}
+
+.form-With-Avatar {
+  height: 100%;
+  width: 200px;
```

Comment:
I think, we do not need this width as we are already mentioning one default width of 400px.

Reply:
yes tithi here width is not required i removed it.